### PR TITLE
[cmp.categories] Replace 'operator admits' phrasing.

### LINKED ISSUE #2357: Wonky wording: operators that "admit" other operators?
E.g. in [[cmp.weakord]/1](http://eel.is/c++draft/cmp.weakord#1.sentence-1):
> The `weak_­ordering` type is typically used as the result type of a three-way comparison operator that (a) admits all of the six two-way comparison operators ([expr.rel], [expr.eq]), and (b) does not imply substitutability.

What does it mean for an operator to "admit" another operator? I can't find a definition of it anywhere. Is there maybe a better way to phrase what is intended here? :)

### PR DESCRIPTION
**Pull request by jensmaurer**
Fixes #2357.

### CODE REVIEW

File: source/support.tex [hunk 1]
```diff
@@ -4283,9 +4283,10 @@
 \pnum
 The \tcode{partial_ordering} type is typically used
 as the result type of a three-way comparison operator\iref{expr.spaceship}
-that (a) admits all of the six two-way comparison operators (\ref{expr.rel}, \ref{expr.eq}),
-(b) does not imply substitutability,
-and (c) permits two values to be incomparable.%
+for a type that is intended to support
+all of the six two-way comparison operators (\ref{expr.rel}, \ref{expr.eq}),
```
Comment (Eelis) [inline source/support.tex]: I think the space before `(\ref` should normally be a `~`.

**Comment by W-E-Brown** [comment]
Sorry, but I fail to see any issue here worth fixing.  My English dictionary defines "admit" as "To be capable <https://en.wiktionary.org/wiki/capable> of; to permit <https://en.wiktionary.org/wiki/permit>", and that makes perfect sense to me in the given context.  Why do we need further definition?

By the way, the proposed rewording introduces some notion of intention, for which I can find no justification in this context.

In short, I'm at a loss to understand what's wrong with the plain English reading of what's in the Draft.

**Comment by jensmaurer** [comment]
@W-E-Brown, the concern here is that "an operator" admits something, but it seems we should talk about the type having certain properties here, not the operator.

File: source/support.tex (hunk 1 above)
Comment (jensmaurer) [reply source/support.tex]: Fixed.

**Comment by jensmaurer** [comment]
Editorial meeting: Postpone due to concurrent LWG work in this area.

File: source/support.tex [hunk 2]
```diff
@@ -4238,9 +4238,10 @@
 \pnum
 The \tcode{partial_ordering} type is typically used
 as the result type of a three-way comparison operator\iref{expr.spaceship}
-that (a) admits all of the six two-way comparison operators (\ref{expr.rel}, \ref{expr.eq}),
-(b) does not imply substitutability,
-and (c) permits two values to be incomparable.
+that is intended to support
```
Comment (jensmaurer) [inline source/support.tex:4241]: add "for a type", consistent with the rest

**Comment by jensmaurer** [comment]
Also drop "intent" if possible.

File: source/support.tex [hunk 3]
```diff
@@ -4395,9 +4395,9 @@
 \pnum
 The \tcode{weak_ordering} type is typically used
 as the result type of a three-way comparison operator\iref{expr.spaceship}
-that (a) admits all of the six
-two-way comparison operators\iref{expr.rel,expr.eq},
-and (b) does not imply substitutability.
+for a type that admits
+all of the six two-way comparison operators\iref{expr.rel,expr.eq}
+and for which equality need not imply substitutability,
```
Comment (CaseyCarter) [inline source/support.tex:4400]:
```suggestion
and for which equality need not imply substitutability.
```

File: source/support.tex [hunk 4]
```diff
@@ -4520,9 +4520,9 @@
 \pnum
 The \tcode{strong_ordering} type is typically used
 as the result type of a three-way comparison operator\iref{expr.spaceship}
-that (a) admits all of the six
-two-way comparison operators\iref{expr.rel,expr.eq},
-and (b) does imply substitutability.
+for a type that admits
+all of the six two-way comparison operators\iref{expr.rel,expr.eq}
+and for which equality does imply substitutability,
```
Comment (CaseyCarter) [inline source/support.tex:4525]:
```suggestion
and for which equality does imply substitutability.
```

**Review by CaseyCarter — CHANGES_REQUESTED** [review]

File: source/support.tex (hunk 3 above)
Comment (jensmaurer) [reply source/support.tex]: fixed

**Review by CaseyCarter — APPROVED** [review]

File: source/support.tex (hunk 3 above)
Comment (CaseyCarter) [reply source/support.tex]: (You need to resolve these comments; only PR authors and folks with commit authority can resolve. Yes, that's right, Github doesn't allow comment authors to resolve their own comments.)

Comment (jensmaurer) [reply source/support.tex]: Yes, but it seems more friendly to the comment author to have him review the unresolved comment back-and-forth before hiding it from view.

**Comment by jensmaurer** [comment]
@tkoeppe, what about this one?

File: source/support.tex (hunk 3 above)
Comment (CaseyCarter) [reply source/support.tex]: That does make sense given that a great many of our commentors are not full-time GitHub residents.

File: source/support.tex (hunk 2 above)
Comment (tkoeppe) [reply source/support.tex]: obsolete
